src/sort.js

```diff
@@ -728,9 +740,15 @@ function getTrailingSpaces(node, sourceCode) {
   return lines[0];
 }
 
+function ensureReactIsFirst(a, b) {
```

Comment:
Maybe we can do the same thing with `styled-components` ? Something like a priority list ["react", "react-dom", "styled-components"] for top level dependencies.

But it's maybe overkill :)